pyproject.toml

```diff
@@ -21,3 +21,53 @@ exclude = '''
                      # the root of the project
 )
 '''
+
+[tool.poetry]
+name = "pinecone"
```

Comment:
Now that I've done more local testing - I think this needs to be `pinecone-client`.